document permissions changes for Puppet gitops (#17367)
fleetdm/fleet: docs/Using Fleet/manage-access.md

[hosts](https://fleetdm.com/docs/using-fleet/rest-api#hosts), [software](https://fleetdm.com/docs/using-fleet/rest-api#software), [queries](https://fleetdm.com/docs/using-fleet/rest-api#queries), and [policies](https://fleetdm.com/docs/using-fleet/rest-api#policies). Check out [the user permissions
table](#user-permissions) above for global user permissions.

> Users can be a member of multiple teams in Fleet.

Reviewer: Let's revert to the old language:

```suggestion
Users can be assigned to multiple teams in Fleet.
```

(We recently updated the UI to change the "team members" terminology to reference users instead, and updated  the docs to match.)

> Users that are members of multiple teams can be assigned different roles for each team. For example, a user can be given access to the "Workstations" team and assigned the "Observer" role. This same user can be given access to the "Servers" team and assigned the "Maintainer" role.

Reviewer: Same here:
```suggestion
Users with access to multiple teams can be assigned different roles for each team. For example, a user can be given access to the "Workstations" team and assigned the "Observer" role. This same user can be given access to the "Servers" team and assigned the "Maintainer" role.
```
